plugins/lando-services/php/php.js

```diff
@@ -147,7 +147,9 @@ module.exports = function(lando) {
         TERM: 'xterm',
         COMPOSER_ALLOW_SUPERUSER: 1,
         PATH: path.join(':'),
-        LANDO_WEBROOT: webroot
+        LANDO_WEBROOT: webroot,
+        BLACKFIRE_CLIENT_ID: process.env.BLACKFIRE_CLIENT_ID,
```

Comment:
These values aren't actually needed in the PHP container either.  You need BLACKFIRE_SERVER_* in the agent container, and BLACKFIRE_CLIENT_* in whatever container runs `blackfire run` or `blackfire curl` (currently, this would have to be the blackfire/blackfire container, since that's the only place the client is installed).